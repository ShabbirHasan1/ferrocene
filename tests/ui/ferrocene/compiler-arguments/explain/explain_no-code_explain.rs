// check-fail
// compile-flags: --explain --explain

fn main() {}

// ferrocene-annotations: um_rustc_explain
