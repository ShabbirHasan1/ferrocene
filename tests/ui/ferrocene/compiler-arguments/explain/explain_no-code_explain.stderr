error: --explain is not a valid error code

